.travis.yml

```diff
@@ -22,7 +22,9 @@ after_success:
     - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
     - export REPO=breerly/hello-server
     - export TAG=`if [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_BRANCH; fi`
-    - docker build -f Dockerfile -t $REPO:$COMMIT .
+    - CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o main .
+    - docker build -f Dockerfile.scratch -t $REPO:$COMMIT .
+    - docker images
     - docker tag $REPO:$COMMIT $REPO:$TAG
     - docker tag $REPO:$COMMIT $REPO:travis-$TRAVIS_BUILD_NUMBER
     - docker push $REPO
```

Comment:
perhaps - the weird part is that the script would depend on the travis environment